client/src/main/java/org/schlunzis/kurtama/client/util/VersionManager.java

```diff
@@ -0,0 +1,21 @@
+package org.schlunzis.kurtama.client.util;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class VersionManager {
+
+    private static final String DEFAULT_VERSION = "0.0.1-alpha+20240808121326";
+
+    public String getVersion() {
+        String version = getClass().getPackage().getImplementationVersion();
+        if (version == null || version.isEmpty())
+            version = DEFAULT_VERSION;
+        return version;
+    }
+
+    public String getDockerVersion() {
```

Comment:
I discussed the idea of a `getFormattedVersion(String formatter)` already early. I assumed, this would remove the need for a specific docker version getter but as the docker version needs to replace each plus with a minus, I can't think of a formatter to do that on the fly....
Maybe not so smart of an idea after all...